src/irr/asset/CGLIWriter.cpp

```diff
@@ -186,13 +195,18 @@ namespace irr
 					const auto layerData = texture.data(gliLayer, gliFace, region->imageSubresource.mipLevel);
 					const auto sourceData = ptrBeginningOfRegion + (layer * layerSize);
```

Comment:
yeah read https://github.com/Crisspl/IrrlichtBAW/pull/21/files/80e64777de0708872d875906cd1439c291307e99..dbf4f204b87c39a090827d50ca0844bd7d3fcb89#r368247906

You need to account for x,y,z offsets in the minimum of a region.